subgraphs/rocket-pool/abis/rocketDAONodeTrusted.json

```diff
@@ -0,0 +1,427 @@
+[
```

Comment:
In an attempt to keep things consistent across the directory, could you make all the abi filenames start with a capital letter?

Reply:
done now